src/components/modals/ActionCreatorEditor.tsx

```diff
@@ -145,7 +144,7 @@ const conditionalEntityTags = (entities: CLM.EntityBase[], actions: CLM.ActionBa
         .map(e => convertEntityToConditionalTags(e))
         .reduce((a, b) => [...a, ...b], [])
 
-    const uniqueConditionTags = [
```

Comment:
I'm not sure which warning was telling you do this this but I'm skeptical as some of these changes which remove helpful information. Even more ironic because you're usually the one to complain about lack of documentation. I do prefer to avoid unnecessary variable names and use `return` when it's a simple object that is self explanatory. It can prevent an extra variable which can become stale as return type changes, but these names added semantic information. People reading the code have to understand, what the operation does now. When it was named "unique" it was clear it reduced it to unique tags and how it reduced it was abstracted away

Reply:
I suggest a comment before the return statement for clarificaiton

Reply:
If the comment is just going to re state what the variable was named then it serves no purpose. Might as well put the variable back. This is why it was explicitly named to begin with.

Reply:
The lint here is "unnecessary local variable".  In these cases, if the function-level comment and return type aren't providing enough information to support eliding the variable name on the returned value, then I think we should adjust the function comment.  Please LMK which instances you feel this is the case for.

Reply:
We should either (1) remove the tslint requirement or (2) add a descriptive comment.   I vote for (2)

Reply:
Cool, took a swag at (2) for this function.